src/js/components/stock-movement-wizard/combined-shipments/AddItemsPage.jsx

```diff
@@ -159,17 +159,49 @@ const FIELDS = {
           },
         }),
       },
-      quantityRequested: {
+      packsRequested: {
         type: TextField,
-        label: 'react.stockMovement.quantity.label',
-        defaultMessage: 'Qty',
+        label: 'react.stockMovement.quantityPerUom.label',
+        defaultMessage: 'Quantity (per UoM)',
```

Comment:
I think this should be something different. "Packs requested" is something different than "quantity per uom" ("quantity per uom" is "pack size")

Reply:
this name was suggested to me by Manon, I don't have any opinion about it so this needs further confirmation